charts/ocis/templates/antivirus/pdb.yaml

```diff
@@ -1,4 +1,5 @@
 {{ if .Values.features.virusscan.enabled }}
 {{- include "ocis.appNames" (dict "scope" . "appName" "appNameAntivirus" "appNameSuffix" "") -}}
+{{- $_ := set . "autoscaling" (default (default (dict) .Values.autoscaling) .Values.services.antivirus.autoscaling) -}}
```

Comment:
This line and all the following need to go into the `hpa.yaml` files.

Reply:
Sorry for the derp, funny how it still worked... Fixing now.